tests/io/test_hysplit.py

```diff
@@ -0,0 +1,17 @@
+import act
+import matplotlib.pyplot as plt
+
+from arm_test_data import DATASETS
+
+
+def test_read_hysplit():
+    filename = DATASETS.fetch('houstonaug300.0summer2010080100')
```

Comment:
Can you add the DATASETS call to the sample_files.py instead and call that here instead for consistency with the other modules?